Example/nRFBlinky/Models/BlinkyPeripheral.swift

```diff
@@ -88,7 +88,7 @@ class BlinkyPeripheral: NSObject, CBPeripheralDelegate {
                 self.post(.blinkyDidDisconnect(self, error: nil))
             }
         }
-        onConnected {
+        _ = onConnected {
```

Comment:
This, and the `onStateChange` observer never gets dispose. But that's not a big deal, as they should live until the app is killed anyway.